.circleci/config.yml

```diff
@@ -573,10 +573,10 @@ workflows:
       - run-test-ios-14:
           xcode_version: '14.0.0'
       - run-test-ios-13:
-          xcode_version: '14.0.0'
+          xcode_version: '13.4.1'
           <<: *release-branches-and-main
       - run-test-ios-12:
-          xcode_version: '14.0.0'
+          xcode_version: '13.4.1'
```

Comment:
Maybe add a comment explaining why we are not using xcode 14?

Reply:
Good poitm